packages/github/README.md

```diff
@@ -55,3 +55,19 @@ const newIssue = await octokit.issues.create({
   body: 'Hello Universe!'
 });
 ```
+
+## Webhook Payload Typescript Definitions
+`@octokit/webhooks` provides complete typescript definitions for the payloads associated with each event. You can cast the provided payload to these types for more type information as needed.
+
+To do this, first install the npm module `npm install --save @octokit/webhooks`
```

Comment:
is `--save` an option? the [docs](https://docs.npmjs.com/cli/install) dont show that option. `--save-prod` is the default so i dont think we need any save flag at all?